### PR TITLE
GlusterFS - tidy up the directories as part of the uninstall playbook

### PR DESCRIPTION
**Pull request by DanyC97**
this PR will delete files and/or directories (not all the paths are directories already but w/o this PR we don't delete them).

Also for consistency changed 2 task's name (will avoid doing that in the future, bad habit)

### CODE REVIEW

**Comment by DanyC97** [comment]
@jarrpa pls review, cheers

**Comment by DanyC97** [comment]
will push a PR to 3.7 branch once https://github.com/openshift/openshift-ansible/issues/6801#issuecomment-366897939 will get in (to avoid any rebase etc )

File: roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: Delete pre-existing heketi resources
+- name: Delete pre-existing Heketi resources
```
Comment (jarrpa) [inline roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: heketi is intentionally all lower-case. :)

Comment (DanyC97) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: okay but then again [this](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml#L67) should be lower-case too? :)

Comment (jarrpa) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: Yup! Not all developers are consistent about it, but officially it is.

Comment (DanyC97) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml]: @jarrpa in my env (3.7 branch) the dir was still present, with the change the dir was gone. And i'll run / put a simple test case and attach the output here so you can see it

**Comment by DanyC97** [comment]
closing as i don't know what on earth i've seen :-( - easy to chase with a v simple test

```
﻿root@ubuntu:~# tree dir_parent/
dir_parent/
├── f1
│   └── f2
└── f3
    └── f4
        └── f5

5 directories, 0 files
root@ubuntu:~# ansible-playbook dir_playbook.yml --tag delete
PLAY [Creating multiple by checking folders] ***************************************************************************************************************************************************************

TASK [Gathering Facts] *************************************************************************************************************************************************************************************
ok: [localhost]

TASK [Delete folders] **************************************************************************************************************************************************************************************
changed: [localhost] => (item=./dir_parent/f1)

PLAY RECAP *************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

root@ubuntu:~# tree dir_parent/
dir_parent/
└── f3
    └── f4
        └── f5

```

and the test playbook was
```
root@ubuntu:~# cat dir_playbook.yml 
---
  - name: Delete folders
    file:
      path: "{{item}}"
      state: absent
    with_items:
    - ./dir_parent/f1
    tags: delete


```

**Pull request closed without merging**